### PR TITLE
Update some stuff...

### PR DESCRIPTION
**Pull request by hassanila97**
Updated to output only the "real output" and renamed...

### CODE REVIEW

**Comment by Andrews54757** [comment]
Please remove the "Fuck" skin

File: src/modules/CommandList.js [hunk 1]
```diff
@@ -191,7 +191,7 @@ gameServer.clients[i].playerTracker.minioncontrol = false;
         var ok = split[1];
         var abort = false;
         if (!fs.existsSync('./packet')) {
-            console.log("[Console] Error: could not preform action. Cause: You deleted folders or you are useing a binary");
+            console.log("[Console] Error: could not preform action. Cause: You deleted folders or you are using a binary");
```
Comment (ItzLevvie) [inline src/modules/CommandList.js]: Should be ```perform``` instead of ``preform```.

Comment (hassanila97) [inline src/modules/CommandList.js]: ltzLEEvie Nice.... fix incoming  thx

**Comment by hassanila97** [comment]
Skin  "fuck"  has been removed now... @Andrews54757

**Pull request merged**